support/langkit_support-generic_api-introspection.ads

```diff
@@ -33,35 +33,52 @@ package Langkit_Support.Generic_API.Introspection is
    -- Polymorphic values --
```

Comment:
I much prefer the API like that myself, thanks for doing that !

Reply:
Great, thanks for the design discussion!